### PR TITLE
Used timeout function instead of undefined clickhouse_client_loop_timeout in functional tests

### PR DESCRIPTION
**Pull request by Demilivor**
Using `clickhouse_client_loop_timeout ` in bash script will raise:
`clickhouse_client_loop_timeout: command not found ` error.

### Changelog category (leave one):
- Build/Testing/Packaging Improvement

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
02242_delete_user_race and 02243_drop_user_grant_race tests have been corrected

### CODE REVIEW

**Comment by Demilivor** [comment]
Good day @tavplubix  could you please add a label 'Can be tested' and be the reviewer for this minor pull request?

File: tests/queries/0_stateless/02243_drop_user_grant_race.sh [hunk 1]
```diff
@@ -29,7 +29,7 @@ function create_drop_grant()
 export -f create_drop_grant
 
 TIMEOUT=10
-clickhouse_client_loop_timeout $TIMEOUT create_drop_grant 2> /dev/null &
+timeout $TIMEOUT bash -c create_drop_grant 2> /dev/null &
```
Comment (Demilivor) [inline tests/queries/0_stateless/02243_drop_user_grant_race.sh:32]: Why do we need timeout here?

**Comment by tavplubix** [comment]
Oops, it was a race condition between two PRs: https://github.com/ClickHouse/ClickHouse/pull/37622 and https://github.com/ClickHouse/ClickHouse/pull/37303

**Comment by robot-clickhouse-ci-1** [comment]
This is an automated comment for commit 0c9af6492af0515b7cb4d72a90d9d41ca7b5874c with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/51923/0c9af6492af0515b7cb4d72a90d9d41ca7b5874c/ci_running.html)
The overall status of the commit is 🟡 pending

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🟡 pending</td></tr>
<tr><td>Mergeable Check</td><td>Checks if all other necessary checks are successful</td><td>🟢 success</td></tr>
</table>

File: tests/queries/0_stateless/02243_drop_user_grant_race.sh (hunk 1 above)
Comment (Demilivor) [reply tests/queries/0_stateless/02243_drop_user_grant_race.sh:33]: Done

**Review by tavplubix — APPROVED** [review]

**Pull request merged**